vllm/_custom_ops.py

```diff
@@ -9,6 +9,11 @@
 
 logger = init_logger(__name__)
 
+# NOTE: 7/26/24: Use Triton implementation of AWQ functions. Currently, only
+#       awq_dequantize is implemented, but awq_gemm coming soon.
+
+use_awq_triton = False
```

Comment:
How does this get set? Should this be an environment variable or based on hardware available?